src/vm/arm64/asmhelpers.asm

```diff
@@ -1326,5 +1329,62 @@ Fail
     LEAF_END
 #endif
 
+;
+; JIT Static access helpers when coreclr host specifies single appdomain flag 
+;
+
+; ------------------------------------------------------------------
+; void* JIT_GetSharedNonGCStaticBase(SIZE_T moduleDomainID, DWORD dwClassDomainID)
+
+    LEAF_ENTRY JIT_GetSharedNonGCStaticBase_SingleAppDomain
+    ; If class is not initialized, bail to C++ helper
+    add x2, x0, #DomainLocalModule__m_pDataBlob
+    ldrb w2, [x2, w1]
```

Comment:
@rahku This is not a valid instruction form.  Do you know why this assembled on Windows Arm64 build?  I am concerned this is as arm64 windows assembler issue.  Also line 1368.  I will change them to 

```
ldrb w2, [x2, x1]
```